js/modules/tamagotchi.js

```diff
@@ -9,10 +9,51 @@ export default class Tamagotchi {
 
   displayHealth = (elementSelector) => {
     const displayElement = document.querySelector(elementSelector);
-    displayElement.innerText = this.energy.value;
+    displayElement.innerText = this.health.value;
   };
 
+  decreaseParameters = () => {
+
+    const healthInterval = setInterval(() => {
+      if (this.hunger.value <= 0 || this.energy.value <= 0) {
+        if (this.health.value > 0) {
+          this.health.value -= 1
+        } else {
+          clearInterval(healthInterval)
+        }
+      }
+    }, 1000)
+
+    const energyInterval = setInterval( () => {
```

Comment:
**nitpick:** We should set intervals as part of class constructor - currently you don't have any control over it. What if you'd like to stop the timer using some other action? It would be impossible in current implementation